src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/AuthDAO.scala

```diff
@@ -18,6 +18,8 @@ trait AuthDAO {
 
   def loadGroupByEmail(groupEmail: String, txn: RawlsTransaction): Option[RawlsGroup]
 
+  def findRelatedWorkspaces(groupRef: RawlsGroupRef, txn: RawlsTransaction): Seq[Workspace]
```

Comment:
Or am I misunderstanding what this does?  Either way, I would like a comment describing it